docs/src/app/pages/Accordion/ExampleNormal.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Accordion } from 'react-lds';
+
+const sections = [
```

Comment:
This seems very verbose. You could implement it via two component: `Accordion` and `AccordionSection` and keep open state in `Accordion`. You could loop over children via `React.Children` and `React.cloneElement` and add open state this way.